src/components/forms/OfficeHoursForm.tsx

```diff
@@ -28,21 +26,24 @@ import { api } from './api';
 import { chakraStyles } from './selectStyles';
 import {
   HOW_DID_YOU_HEAR_ABOUT_ESP_OPTIONS,
-  OTHER,
   PROJECT_CATEGORY_OPTIONS,
   TEAM,
-  TIMEZONE_OPTIONS
+  PROJECT_FEEDBACK,
+  TIMEZONE_OPTIONS,
 } from './constants';
 import { OFFICE_HOURS_THANK_YOU_PAGE_URL, TOAST_OPTIONS } from '../../constants';
 
-import { IndividualOrTeam, OfficeHoursFormData, ReasonForMeeting } from '../../types';
+import { IndividualOrTeam, OfficeHoursFormData, OfficeHoursRequest } from '../../types';
 
 export const OfficeHoursForm: FC = () => {
   const [individualOrTeam, setIndividualOrTeam] = useState<IndividualOrTeam>('Individual');
-  const [reasonForMeeting, setReasonForMeeting] = useState<ReasonForMeeting>(['']);
+  const [officeHoursRequest, setOfficeHoursRequest] = useState<OfficeHoursRequest>('Advice');
   const router = useRouter();
   const toast = useToast();
 
+  const isTeam = individualOrTeam === TEAM
+  const requestingFeedback = officeHoursRequest === PROJECT_FEEDBACK
```

Comment:
maybe this could be renamed to `requestingProjectFeedback` to be extra clear and avoid any confusion (advice can also count as feedback)